README.md

```diff
@@ -121,11 +168,16 @@ let package = Package(
   ]
 )
 ```
+And then build your app or example app:
+
+```bash
+swift build
+```
```

Comment:
There's a section called "### Building and running" listing a very similar thing

Reply:
... except its about using a baked-in example app thats only  discoverable if you've cloned a local copy and ferret around to find it..and run conmands in the clone, not if u are building your own app with AppKid as a depenancy. I see there's a swift command in your repo to build it, but most ppl are gonna just want to use it via a dependency url, not a local clone